pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -461,6 +465,47 @@ async def publish_client(self, working_dir, from_release_tag, release_name, buil
         # Publish the clients to our S3 bucket.
         await exectools.cmd_assert_async(f"aws s3 sync --no-progress --exact-timestamps {base_to_mirror_dir}/{build_arch} s3://art-srv-enterprise/pub/openshift-v4/{build_arch}", stdout=sys.stderr)
 
+    def publish_baremetal_installer_binary(self, from_release_tag: str, client_mirror_dir: str):
+        release_pullspec = f'{constants.QUAY_RELEASE_REPO_URL}:{from_release_tag}'
+        image_stat, baremetal_installer_pullspec = get_release_image_pullspec(release_pullspec, 'baremetal-installer')
+
+        if image_stat:  # != 0, image does not exist
+            self._logger.error('Error getting baremetal-installer binary from %s', release_pullspec)
+            return
+
+        # Get baremetal image pullspec
+        res, image_info = get_release_image_info_from_pullspec(baremetal_installer_pullspec)
+        if res:
+            self._logger.error('Error retrieving image info for %s', baremetal_installer_pullspec)
+            return
+
+        # Check rhel version (used for archive naming
```

Comment:
in the ticket description, I thought it should always be rhel8, maybe I am wrong, can you double confirm?

Reply:
According to [this comment](https://issues.redhat.com/browse/ART-7207?focusedId=22586790&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-22586790), we decided to hardcode the rhel8 suffix into the tarball name. This will eventually change once baremetal-installer moves to rhel9. I'll add a comment in `ocp-build-data` as a reminder